clearpath_generator_common/clearpath_generator_common/param/platform.py

```diff
@@ -236,6 +236,28 @@ def generate_parameters(self, use_sim_time: bool = False) -> None:
                                     ],
                                     'contains': ['MCU']}}}}})
 
+            # Add cooling for A300 only for now
+            if self.clearpath_config.get_platform_model() == Platform.A300:
+                self.param_file.update(
+                    {self.DIAGNOSTIC_AGGREGATOR_NODE: {
+                        'platform': {
+                            'analyzers': {
+                                'cooling': {
+                                    'type': 'diagnostic_aggregator/GenericAnalyzer',
+                                    'path': 'Cooling',
+                                    'contains': ['Fan', 'Thermal']}}}}})
+
+            if self.clearpath_config.platform.enable_ekf:
+                self.param_file.update(
+                    {self.DIAGNOSTIC_AGGREGATOR_NODE: {
+                        'platform': {
+                            'analyzers': {
+                                'odometry': {
+                                    'expected': [
+                                        'ekf_node: Filter diagnostic updater',
+                                        'ekf_node: odometry/filtered topic status',
+                                    ]}}}}})
```

Comment:
Same here with regards to splitting the closing characters across multiple lines